spec.html

```diff
@@ -24191,15 +24701,16 @@ <h2>Syntax</h2>
         ClassElementList[?Yield, ?Await] ClassElement[?Yield, ?Await]
 
       ClassElement[Yield, Await] :
-        MethodDefinition[?Yield, ?Await]
-        `static` MethodDefinition[?Yield, ?Await]
-        FieldDefinition[?Yield, ?Await] `;`
-        `static` FieldDefinition[?Yield, ?Await] `;`
+        DecoratorList[?Yield, ?Await]? MethodDefinition[?Yield, ?Await]
+        DecoratorList[?Yield, ?Await]? `static` MethodDefinition[?Yield, ?Await]
+        DecoratorList[?Yield, ?Await]? FieldDefinition[?Yield, ?Await] `;`
+        DecoratorList[?Yield, ?Await]? `static` FieldDefinition[?Yield, ?Await] `;`
         ClassStaticBlock
         `;`
 
       FieldDefinition[Yield, Await] :
         ClassElementName[?Yield, ?Await] Initializer[+In, ?Yield, ?Await]?
+        `accessor` ClassElementName[?Yield, ?Await] Initializer[+In, ?Yield, ?Await]?
```

Comment:
Perhaps there's an issue covering this, but `accessor` is already a valid class element name so the following is valid grammar today:

```js
class Foo {
  accessor
  x = 1
}
```

Does this need `no [LineTerminator] here`?